[2024-10-16T09:12:31.965+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-10-10T00:00:00+00:00 [queued]>
[2024-10-16T09:12:31.979+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-10-10T00:00:00+00:00 [queued]>
[2024-10-16T09:12:31.980+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:12:31.980+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:12:31.981+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:12:31.997+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-10-10 00:00:00+00:00
[2024-10-16T09:12:32.002+0000] {standard_task_runner.py:55} INFO - Started process 435 to run task
[2024-10-16T09:12:32.009+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-10-10T00:00:00+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmp_kr77g5s']
[2024-10-16T09:12:32.011+0000] {standard_task_runner.py:83} INFO - Job 88: Subtask insert_rate
[2024-10-16T09:12:32.204+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-10-10T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:12:32.263+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:12:32.332+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-10-10T00:00:00+00:00
[2024-10-16T09:12:32.334+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 492.66863375, '2024-10-10')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:12:32.346+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:12:32.613+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:12:32.627+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 492.66863375, '2024-10-10')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:12:32.630+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:12:32.644+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20241010T000000, start_date=20241016T091231, end_date=20241016T091232
[2024-10-16T09:12:32.697+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:12:32.759+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T10:07:57.045+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-10-10T00:00:00+00:00 [queued]>
[2024-10-16T10:07:57.059+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-10-10T00:00:00+00:00 [queued]>
[2024-10-16T10:07:57.060+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:07:57.061+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T10:07:57.062+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:07:57.086+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-10-10 00:00:00+00:00
[2024-10-16T10:07:57.097+0000] {standard_task_runner.py:55} INFO - Started process 12317 to run task
[2024-10-16T10:07:57.103+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-10-10T00:00:00+00:00', '--job-id', '2878', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmpu83htr_a']
[2024-10-16T10:07:57.104+0000] {standard_task_runner.py:83} INFO - Job 2878: Subtask insert_rate
[2024-10-16T10:07:57.213+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-10-10T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T10:07:57.273+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T10:07:57.351+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-10-10T00:00:00+00:00
[2024-10-16T10:07:57.352+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 492.66863375, '2024-10-10')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T10:07:57.364+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:07:57.605+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:07:57.614+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 492.66863375, '2024-10-10')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T10:07:57.618+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T10:07:57.630+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20241010T000000, start_date=20241016T100757, end_date=20241016T100757
[2024-10-16T10:07:57.651+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T10:07:57.674+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
